src/index.ts

```diff
@@ -95,38 +104,45 @@ export const makeStore = <T extends State>(initialState: T): Store<T> => {
       return changes
     }, {})
 
-  return {
-    get state() {
-      return state
-    },
-
-    set: (incoming, forceNotify = false) => {
-      /* If the argument is a function, run it */
-      const incomingState = incoming instanceof Function ? incoming(state) : incoming
+  const _set: Store<T>["set"] = (incoming, options) => {
```

Comment:
Is there a specific reason for this renaming?

Reply:
yes, so I put it to a separate variable, so accessing it does not depend on the order of the variables. I could also inline it in the return statement, but I felt it's more clean that way